cli/src/stake.rs

```diff
@@ -1892,12 +1892,20 @@ pub fn process_split_stake(
             return Err(CliError::BadParameter(err_msg).into());
         }
 
-        let minimum_balance =
-            rpc_client.get_minimum_balance_for_rent_exemption(StakeState::size_of())?;
-
+        let minimum_balance = rpc_client
+            .get_minimum_balance_for_rent_exemption(StakeState::size_of())?
+            + rpc_client.get_stake_minimum_delegation()?;
```

Comment:
I took the view of putting the check in place when creating the vote account for two reasons:

- there was already a rent balance check there, and the lamports variable is already in scope, removing the need for an rpc balance check
- it stops the user earlier on, rather than creating a stake account, discovering they now can't do anything with it and being stuck with a useless stake account which they then need to close (or top up).

However this is putting the cart in front of the horse a bit, my first pass had the check in the delegate section and not create, do you think we should move it back there then?

Reply:
@brooksprumo sorry to bother, following up on my earlier comment above, should I move the check into the Delegate section as this aligns better with what the on chain program enforces?

Reply:
Why not use --force? This exists in the delegate command already to bypass the vote account not having a root slot etc, keeps the flags consistent